src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Receiving shared data
+subhead: Sharing on mobile made simple
+authors:
+  - petelepage
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  On a mobile device, sharing should be as simple as clicking the Share button,
+  choosing an app, then choosing who to share with. The Web Share Target API
+  allows installed web apps to register with the underlying operating system to receive shared content.
+tags:
+  - post
+  - capabilities
+---
+
+On a mobile device, sharing should be as simple as clicking the **Share** button,
+choosing an app, and choosing who to share with. For example, you may want to
+share an interesting article, either by emailing it to friends or tweeting it to
+the world.
+
+In the past, only native apps could register with the operating system to
+receive shares from other installed apps. But with the Web Share Target API,
+installed web apps can register with the underlying operating system
+as a share target to receive shared content.
+Support for text and data was added in Chrome 71, and
+support for files was added in Chrome 76.
+
+{% Aside %}
+The Web Share Target API is only half of the magic. Web apps can share data,
+files, links, or text using the Web Share API. See
+[Share like a native](/share-like-a-native) for details.
+{% endAside %}
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 400px;" alt="Android phone with the 'Share via' drawer open."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    System-level share target picker with an installed PWA as an option.
+  </figcaption>
+</figure>
+
+## See Web Share Target in action
+
+1. Using Chrome 76 or later, open the [Web Share Target demo][demo].
+2. When prompted, click **Install** to add the app to your home screen, or
+   use the Chrome menu to add it to your home screen.
+3. Open any app that includes a native share intent, or use the Share button
+   in the demo app.
+4. From the target picker, choose **Web Share Test**.
+
+After sharing to the demo app, you should see all of the information sent via
+the web share target web app.
```

Comment:
This sentence doesn't make sense to me: the first phrase says you're sharing to the demo app, while the main clause says you're sharing via the same app. (Or, at least, I'm assuming "demo app" and "web share target web app" are the same thing.)

Reply:
Fixed.